src/gui/static/src/app/app.config.ts

```diff
@@ -1,15 +1,74 @@
 export const AppConfig = {
+
+  // General settings.
+  ////////////////////////////////
+
+  /**
+   * If true, the option for buying coins via the OTC service will be enabled.
+   * Note: not updated in long time, most likely it won't work.
+   */
   otcEnabled: false,
+  /**
+   * How many addresses the hw wallet can have.
+   */
   maxHardwareWalletAddresses: 1,
+  /**
+   * Max gap of unused addresses a wallet can have between 2 used addresses before the user is
+   * alerted about potential problems that could appear for restoring all addresses when loading
+   * the wallet again using the seed.
+   */
+  maxAddressesGap: 20,
+  /**
+   * ID of the coin on the coin price service. If null, the wallet will not show the USD price.
+   */
+  priceApiId: 'sky-skycoin',
+  /**
+   * This wallet uses the Skycoin URI Specification (based on BIP-21) when creating QR codes and
+   * requesting coins. This variable defines the prefix that will be used for creating QR codes
+   * and URLs. IT MUST BE UNIQUE FOR EACH COIN.
+   */
+  uriSpecificatioPrefix: 'skycoin',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skycoin/pullRequest?prid=7163600)
```suggestion
  uriSpecificatioPrefix: "skycoin",
```